src/launchpad/size/cli.py

```diff
@@ -0,0 +1,60 @@
+from contextlib import ExitStack
+from pathlib import Path
+from typing import TextIO
+
+import click
+from rich.progress import Progress, SpinnerColumn, TextColumn
+
+from ..utils import console, setup_logging
+from .analysis import do_size_analysis
+
+
+@click.command(name="size")
+@click.argument("input_path", type=click.Path(exists=True, path_type=Path), metavar="INPUT_PATH")
+@click.option(
```

Comment:
We lost `output_format` here, which I personally really liked and leveraged alot. See my comments [here](https://github.com/getsentry/launchpad/pull/52/files#r2157657809) and [here](https://github.com/getsentry/launchpad/pull/52/files#r2157661579) about suggestions to return the results and allow the caller to determine what to do with them.